lecli/cli.py

```diff
@@ -59,6 +60,11 @@ def delete():
     pass
 
 
+@cli.group()
+def tail():
+    """Tail logs"""
+    pass
```

Comment:
this isn't complete actually, its there for registering that group with `click` context